app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -140,6 +140,33 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testOnboardingFragment_DefaultSlideTitleDescription_hasCorrectContentDescription() {
+    launch(OnboardingActivity::class.java).use {
+      onView(withId(R.id.onboarding_layout)).check(
+        matches(
+          withContentDescription(
+            "Welcome to Oppia! Learn anything you want in an effective and enjoyable way."
+          )
+        )
+      )
+    }
+  }
+
+  @Test
+  fun testOnboardingFragment_configChange_DefaultSlideTitleDescription_hasCorrectContentDescription() { // ktlint-disable max-line-length
```

Comment:
don't use `// ktlint-disable max-line-length` 
rather try to keep the function name short and understandable.

Reply:
Check the note - https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#how-to-fix-the-most-common-issues

Reply:
Got it, just let me know whether the name similar to what I mentioned in the above comment is valid or not. If it's valid then this linting issues will get solved.

Reply:
commented above.

Reply:
Changes Done